piccolo_api/crud/endpoints.py

```diff
@@ -529,6 +543,27 @@ async def get_all(
         else:
             query = self.table.select(exclude_secrets=self.exclude_secrets)
 
+        visible_fields = split_params.visible_fields
+        if visible_fields:
+            all_columns_map: t.Dict[str, Column] = {
+                key: value
+                for key, value in zip(
+                    [i._meta.name for i in self.table._meta.columns],
+                    self.table._meta.columns,
+                )
+            }
+
+            columns_visible: t.List[Column] = []
+            columns_non_visible: t.List[Column] = []
+
+            for key, value in all_columns_map.items():
+                if key in visible_fields.split(","):
```

Comment:
It would be more efficient to move `visible_fields.split` out of the for loop.

Reply:
You are right. I'll change this to
```python
visible_fields_split: t.List[str] = visible_fields.split(",")

for key, value in all_columns_map.items():
    if key in visible_fields_split:
        columns_visible.append(value)
    else:
        columns_non_visible.append(value)
```